lib/process-services/src/lib/task-list/components/task-form/task-form.component.ts

```diff
@@ -272,15 +274,26 @@ export class TaskFormComponent implements OnInit {
   }
 
   isReadOnlyForm(): boolean {
-      return this.internalReadOnlyForm || !(this.isAssignedToMe() || this.canInitiatorComplete());
+    let readOnlyForm: boolean;
+    if (this.isCandidateMember()) {
```

Comment:
I don't see any unit test related to this part. do we have it ?

Reply:
we have unit test for this case, here is the unit test https://github.com/Alfresco/alfresco-ng2-components/pull/5780/files#diff-33411bc1a3aec7d706037f6f3a5cec1fR166